docs/core/additional-tools/dotnet-svcutil-guide.md

```diff
@@ -0,0 +1,100 @@
+
+The **dotnet-svcutil** tool is an alternative option to the [**WCF Web Service Reference**](/dotnet/core/additional-tools/wcf-web-service-reference-guide) Visual Studio connected service provider which first shipped with Visual Studio 2017 v15.5. The **dotnet-svcutil** tool as a .NET Core CLI tool, is available cross-platform on Linux, MacOS, and Windows.
+
+> [!IMPORTANT]
+> You should only reference services from a trusted source. Adding references from an untrusted source may compromise security. 
+
+## Prerequisites
+
+* [.NET Core SDK](https://www.microsoft.com/net/download) v1.0.4 or later versions
+
+## Getting Started
+
+The following example walks you through the steps required to add a web service reference to a .NET Core console project and invoke the service.  You will create a .NET Core console application named _HelloSvcutil_ and will add a reference to a web service that implements the following contract:
+```csharp
+[ServiceContract]
+public interface ISayHello
+{
+    [OperationContract]
+    string Hello(string name);
+}
+```
+For this example, the web service address will be the following: _http://myhost/SayHello.svc_
+
+From a command window perform the following steps:
+
+1. Create a directory named _HelloSvcutil_ for your project and make it your current directory, as in the following example:
+```console
+mkdir HelloSvcutil
+cd HelloSvcutil
+```
+
+2. From the command window, type [dotnet new console](../tools/dotnet-new.md) to create a new C# console project in that directory, as in the following example:
+```console
+dotnet new console
+```
+
+3. Edit the *HelloSvcutil.csproj* project file and add the [`dotnet-svcutil` NuGet package](https://nuget.org/packages/dotnet-svcutil) as a CLI tool reference, using the following code:
+```xml
+<ItemGroup>
+  <DotNetCliToolReference Include="dotnet-svcutil" Version="1.0.0" />
+</ItemGroup>
+```
+
+4. Restore the project in order to invoke the _dotnet-svcutil_ tool
+```console
+dotnet restore
+```
+
+5. Run _dotnet_ with the _svcutil_ command to generate the web service reference file which will be saved as _HelloSvcutil/ServiceReference1/Reference.cs_
+```console
+dotnet svcutil http://myhost/SayHello.svc
+```
+
+6. Restore the WCF packages for the project
+```console
+dotnet restore
+```
+
+7. Open the `Program.cs` file in your editor, edit the `Main()` method, and replace the auto-generated code with the following code to invoke the web service:
+```csharp
+static void Main(string[] args)
+{
+    var client = new SayHelloClient();
+    Console.WriteLine(client.HelloAsync("dotnet-svcutil").Result);
+}
+```
+
+8. Run the application via [`dotnet run`](../tools/dotnet-run.md).
+```console
+dotnet run
+"Hello dotnet-svcutil!"
```

Comment:
As this is the output, it should be moved out of  the command example, with a sentence like, "You should see the following output:"

Reply:
sure